### PR TITLE
Editor Changes for Role landing pages for issue #87

### PR DESCRIPTION
**Pull request by OHaiDanny**
@nonowazu in this PR i've added the editor changes for roles. I hope I understood the task correctly.. #87 

Couple clarifying things:

1. The names of the roles should be in plural form?
2. the role for melee, ranged, caster dps should be identifed as `dps`?

### CODE REVIEW

File: exampleSite/content/jobs/ranged/_index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+menu:
+  main:
+    name: Ranged
+    parent: jobs
+role: dps
```
Comment (nonowazu) [inline exampleSite/content/jobs/ranged/_index.md:6]: I actually think this is `role: ranged` – at least, that's what I've got working with production content. Double-check with @Roxas92 and @ScrubbyXIV though

File: exampleSite/static/admin/config.yml [hunk 2]
```diff
@@ -50,6 +50,70 @@ collections:
...
+          name: "menu"
+          widget: "hidden"
+          default:
+            main:
+              name: "Casters"
+              parent: "jobs"
+        - label: "Role"
+          name: "role"
+          widget: "hidden"
+          default: "caster"
```
Comment (nonowazu) [inline exampleSite/static/admin/config.yml:116]: Just to make 100% sure – you used the editor to generate the other `_index.md` files as part of this PR?

**Review by nonowazu — CHANGES_REQUESTED** [review]

File: exampleSite/static/admin/config.yml (hunk 2 above)
Comment (OHaiDanny) [reply exampleSite/static/admin/config.yml:116]: Yes that is correct :)

File: exampleSite/content/jobs/ranged/_index.md (hunk 1 above)
Comment (OHaiDanny) [reply exampleSite/content/jobs/ranged/_index.md:6]: Ok let me take it away and check

File: exampleSite/static/admin/config.yml (hunk 2 above)
Comment (OHaiDanny) [reply exampleSite/static/admin/config.yml:116]: actually i'm confusing myself a bit with the Job landing page one. Scratch my previous comment. I didn't check it. Let me do that now.

Comment (OHaiDanny) [reply exampleSite/static/admin/config.yml:116]: ugh these hidden labels again. Let me try to set them up so that they are visible

Comment (nonowazu) [reply exampleSite/static/admin/config.yml]: They're tricky beasts. That's why I thought just going with selections with the value hardcoded might be best.

**Comment by OHaiDanny** [comment]
@nonowazu I've updated it so that a brand new role being added and saved in the CMS will generate the heirarchy in each `_index.md`.

Also adjusted the role name after chatting with Scrubby.

File: exampleSite/static/admin/config.yml (hunk 2 above)
Comment (OHaiDanny) [reply exampleSite/static/admin/config.yml]: Yep! i've turned them into selections in my recent updates. Feel free to review it again @nonowazu

File: exampleSite/content/jobs/melee/_index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+menu:
+  main:
+    name: Melee
+    parent: melee
+role: melee
+---
+Melee do positional and stuff.
```
Comment (nonowazu) [inline exampleSite/content/jobs/melee/_index.md:8]: uhhh, I notice there's a `melee/_index.md` and `melees/_index.md` here.

**Review by nonowazu — CHANGES_REQUESTED** [review]
I think we're good, just a minor, *minor* nit

File: exampleSite/content/jobs/melee/_index.md (hunk 3 above)
Comment (OHaiDanny) [reply exampleSite/content/jobs/melee/_index.md:8]: oh shoot! fixing it right away

Comment (OHaiDanny) [reply exampleSite/content/jobs/melee/_index.md:8]: prolly was generated when I was testing and I forgot to delete it.

Comment (OHaiDanny) [reply exampleSite/content/jobs/melee/_index.md:8]: Ok I removed `melees` since we are going with `melee`

**Review by nonowazu — APPROVED** [review]

**Pull request merged**